### DISCUSSION TITLE
Allure Step Annotation Doesn't Work on ubuntu 22.04 and Windows 11

### DISCUSSION THREAD

**Discussion by mourav2222** [allure-framework/allure2#2035] [Questions & Support]
### Describe the Bug

https://github.com/allure-examples/allure-junit5-gradle.git
IntelliJ IDEA 2023.1.2 (Community Edition)

The @Step annotations are missing in the Allure Report. See attchments

### Steps to Reproduce

1. I installed IntelliJ IDEA 2023.1.2 (Community Edition)
and import the project allure-examples/allure-junit5-gradle.git
2. Execute AllureStepTest from IDE direct (Run...) and 
3. Run gradlew allureReport and allureServe

### Expected Behaviour

Step log: 'Parent annotated step with parameter xxxx' in the allure report

![image](https://github.com/allure-framework/allure2/assets/76436807/e9901c0c-7ea6-403e-a6ee-545816464f32)

### Screenshots or Additional Context

![image](https://github.com/allure-framework/allure2/assets/76436807/35cadfa8-9edb-4293-b7ba-af3588e0d028)

![image](https://github.com/allure-framework/allure2/assets/76436807/e4c263c0-ade8-4f7f-9962-d32c2ce1b8c9)

![image](https://github.com/allure-framework/allure2/assets/76436807/5cc80cac-9231-4ece-ae20-787ea6b63d27)

![image](https://github.com/allure-framework/allure2/assets/76436807/5c19d721-27d4-44b6-a09f-575d0787d770)

### What Language are you using?

Java

### What Framework/Allure Integration you are using?

allure-junit5 v5.6.2

### What version of Allure Integration you are using?

2.8.1

### What version of Allure Report you are using?

2.13.6

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by baev** [comment]
<img width="199" alt="Screenshot 2023-06-19 at 11 42 30" src="https://github.com/allure-framework/allure2/assets/2149631/244b7a02-e72d-403f-8ea4-b8466e549377">

`ajcore` files indicate that AspectJ has some errors. Most likely, you need to update AspectJ dependencies to the latest versions

**Comment by mourav2222** [comment]
@baev  Can you check in this project https://github.com/mourav2222/tests-cmd.git, branch windows, 
Aannotation <code>@Step</code> ist not work in function test1(), Class GoogleTestCase 
Start: GoogleTestCase -> test1()
gradlew allureServe
Why don't work @Step here? 

![image](https://github.com/allure-framework/allure2/assets/76436807/9a3bdc12-9bdb-466a-afe1-77bd4c27120c)

It don't work yet over the main function in the Class Tester.  

![image](https://github.com/allure-framework/allure2/assets/76436807/e9f65593-a128-4576-8f1c-923ae214e929)

I have the project in the company
There the Step annotation works in the direkt start Testfunction in the IDE, but over main function the same arro like previously decribed.

![image](https://github.com/allure-framework/allure2/assets/76436807/6f12582e-6067-4074-9f42-1acf4a30cc0c)

![image](https://github.com/allure-framework/allure2/assets/76436807/55776654-d0a9-4d3d-8b9a-c6b6b140d643)

**Comment by mourav2222** [comment]
@baev Did you verificate with my project https://github.com/mourav2222/tests-cmd.git, branch windows? If i ran over junit5 launcher (to use later on the command line like java -jar <gutester.jar> <options>) all _@Step_ annotations are missing in the report.

**Reply by mourav2222** [reply]
It works, it was my error.

![image](https://github.com/allure-framework/allure2/assets/76436807/dbaf14b4-b48c-4521-adbe-b49fa2fc8fb2)
